Develop

alwan1989 opened the pull request:
update

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/PaddlePaddle/models?pullRequest=4897) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/PaddlePaddle/models?pullRequest=4897) before we can accept your contribution.<br/>**5** out of **19** committers have signed the CLA.<br/><br/>:white_check_mark: ZHUI<br/>:white_check_mark: wawltor<br/>:white_check_mark: aigcliu<br/>:white_check_mark: TingquanGao<br/>:white_check_mark: dyning<br/>:x: Steffy-zxf<br/>:x: guoshengCS<br/>:x: ZeyuChen<br/>:x: FrostML<br/>:x: xiemoyuan<br/>:x: kinghuin<br/>:x: huangjun12<br/>:x: KPatr1ck<br/>:x: ranchlai<br/>:x: Intsigstephon<br/>:x: wuhuachaocoding<br/>:x: littletomatodonkey<br/>:x: godfanmiao<br/>:x: LiuChiachi<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/PaddlePaddle/models?pullRequest=4897) it.</sub>

FrostML committed:
fix predict (#4985)

smallv0221 committed:
Add __len__ to squad dataset (#4989)

* update lrscheduler

* minor fix

* add pre-commit

* minor fix

* Add __len__ to squad dataset

* minor fix

FrostML committed:
update readme of transformer benchmark (#4997)

ZeyuChen committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/models into develop

ZeyuChen committed:
update PaddleNLP README

wawltor committed:
Add the support of amp for the fleet train (#4996)

* add the support of amp for the fleet train
* remove the attribute for the amp

ZeyuChen committed:
remove useless predict_classifier.py

ZeyuChen committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/models into develop

windstamp committed:
updates data format for model SE_ResNet50_vd (#4943)

* add amp support for model SE_ResNet50_vd

* optimize code, add model ResNet50 and SE_ResNet50_vd to list

* merge develop

ZeyuChen committed:
update readme

Steffy-zxf committed:
add model api for paddlenlp (#4998)

* add model api

* update codes

* update codes

* update codes

LiuChiachi committed:
Add readme for metrics (#4999)

wawltor committed:
use paddlenlp downloader for the tokenizer (#5003)

use paddlenlp downloader for the tokenizer

Commit:
add electra pretrain and modify style of electra modeling (#4990)

* add electra pretrain and modify style of electra modeling

* add electra pretrain, modify style of electra modeling and fix problems of review

* delete predict_classifer

* modify accu to acc

* add paddlenlp.metrics.glue

zhiqiu committed:
add custom op gelu for bert amp training (#5008)

* add custom op gelu

* refine run_pretrain

LiuChiachi committed:
Update paddlenlp.metrics README (#5010)

* Update paddlenlp.metrics README

* Add info for Mcc

* delete DuReader metric

* update task name, add API col

joey12300 committed:
Add TokenEmbedding (#4983)

* Add TokenEmbedding

* download corpus embedding data
* load embedding data by specifying corpus name
* extend the vocab of tokenizer from corpus embedding data

* add unk token setting

* modify tokenizer

* add extend voacb

* move jieba tokenizer and rename corpus_name->embedding_name

* use bos url instead of localhost

* add log when loading data

* add token dot computation; add __repr__ of TokenEmbedding

* add color logging

* use paddlenlp.utils.log

* adjust repr

* update pretrained embedding table

* fix padding idx

LiuChiachi committed:
Update perplexity, inherited from Metric (#4995)

* update ppl, inherited from Metric, suitable for common mt and lm task.

* fix ppl bugs, delete useless argument

* fix ppl bugs, return a num not a numpy array.

xiemoyuan committed:
Unified the task name of DGU with paddle1.8 (#5011)

* Unified the task name with paddle1.8

* fixed bug.

smallv0221 committed:
Add DuReader yesno and robust (#4992)

* update lrscheduler

* minor fix

* add pre-commit

* minor fix

* Add __len__ to squad dataset

* minor fix

* Add dureader robust prototype

* dataset implement

* minor fix

* fix var name

* add dureader-yesno train script and dataset

* add readme and fix md5sum

* integrete dureader datasets

smallv0221 committed:
Update datasets naming style (#5014)

* update lrscheduler

* minor fix

* add pre-commit

* minor fix

* Add __len__ to squad dataset

* minor fix

* Add dureader robust prototype

* dataset implement

* minor fix

* fix var name

* add dureader-yesno train script and dataset

* add readme and fix md5sum

* integrete dureader datasets

* change var names: segment to mode, root to data_file

* minor fix

* update var name

kinghuin committed:
Optimize BigruCRF example (#5017)

* optimize lac

* formatted

* optimize lac

* optimize lac

wawltor committed:
update the readme for the benchmark (#5020)

* update the readme for the benchmark

* udpate README file for the bert benchmark

* uddate the readme format for the benchmark bert

Co-authored-by: wawltor <fanzeyang0904@hotmail.com>
Co-authored-by: fangzeyang <fangzeyang@baidu.com>

smallv0221 committed:
Fix dureader api bugs (#5021)

* update lrscheduler

* minor fix

* add pre-commit

* minor fix

* Add __len__ to squad dataset

* minor fix

* Add dureader robust prototype

* dataset implement

* minor fix

* fix var name

* add dureader-yesno train script and dataset

* add readme and fix md5sum

* integrete dureader datasets

* change var names: segment to mode, root to data_file

* minor fix

* update var name

* Fix api bugs

xiemoyuan committed:
fixed DGU typos. (#5018)

Steffy-zxf committed:
Add Sentence Transformer for text matching and Add readme (#5004)

* update docs

* add sbert

* add readme

* update readme

* update codes

wawltor committed:
remove the benchmark score for the bert (#5023)

Co-authored-by: wawltor <fanzeyang0904@hotmail.com>

LemonNoel committed:
Add express task (#5024)

* Add Express Example

* Add Express Data

* Add Ernie for Express Example

* add the express for the paddlenlp

Co-authored-by: wanghuijuan03 <wanghuijuan03@baidu.com>

LemonNoel committed:
Add new express task (#5025)

* Add Express Example

* Add Express Data

* Add Ernie for Express Example

* add the express for the paddlenlp

* remove the ernie_crf file

Co-authored-by: wanghuijuan03 <wanghuijuan03@baidu.com>

ZeyuChen committed:
remove run_ernie_crf.py

LiuChiachi committed:
Update metric bleu and rouge-l, inherited from Metric (#4982)

* update rouge-l, move default_trans_func to utils

* fix error info bugs

* add score method for metric bleu and rouge-l

* update bleu doc

ZeyuChen committed:
add api usage docs

ZeyuChen committed:
add tutorial aistudio account

LiuChiachi committed:
Update SamplerHelper batch, Update TranslationDataset (#4991)

* update SamplerHelper.batch

* update batch_size, delete a useless log

* update SamplerHelper.batch

* update SamplerHelper.batch, change cmp_fn arg to key

* Update TranslationDataset and IWSLT

* fix doc error

ZeyuChen committed:
remove uselss README

kinghuin committed:
fix lac typo and image url (#5028)

ZHUI committed:
Add transformer api doc (#5029)

* update transformer doc

* Update transformers.md

* Update transformers.md

Steffy-zxf committed:
add paddle.models api reference docs (#5030)

* add paddle.models api reference docs

* update docs

* update docs

Commit:
unify run_glue in bert, electra and so on (#5026)

wawltor committed:
Add the model zoo readme for the paddlenlp (#5032)

Co-authored-by: wawltor <fanzeyang0904@hotmail.com>

joey12300 committed:
Optimize TokenEmbedding (#5022)

* 1. move PAD to the end
2. Add Readme.md

* optimize readme doc

Steffy-zxf committed:
Update text matching docs (#5033)

* add paddle.models api reference docs

* update docs

* update docs

* update docs

smallv0221 committed:
Add datasets readme (#5031)

* update lrscheduler

* minor fix

* add pre-commit

* minor fix

* Add __len__ to squad dataset

* minor fix

* Add dureader robust prototype

* dataset implement

* minor fix

* fix var name

* add dureader-yesno train script and dataset

* add readme and fix md5sum

* integrete dureader datasets

* change var names: segment to mode, root to data_file

* minor fix

* update var name

* Fix api bugs

* add dataset readme

* add express ner

* update readme format

ZeyuChen committed:
add English Readme

ZeyuChen committed:
update examples README

smallv0221 committed:
Change dataset doc path (#5034)

* update lrscheduler

* minor fix

* add pre-commit

* minor fix

* Add __len__ to squad dataset

* minor fix

* Add dureader robust prototype

* dataset implement

* minor fix

* fix var name

* add dureader-yesno train script and dataset

* add readme and fix md5sum

* integrete dureader datasets

* change var names: segment to mode, root to data_file

* minor fix

* update var name

* Fix api bugs

* add dataset readme

* add express ner

* update readme format

* fix format bug

* change readme path

* fix format bug

ZeyuChen committed:
update README.md

LiuChiachi committed:
Add couplet examples (#5007)

* add couplet

* simplify model code

* simplify code

* update couplet README

* add pad_token to TranslationDataset, update CoupletDataset

* update couplet url, add couplet generation example

* update TranslationDataset

* upadte classname to self in __init__

* update README.md

smallv0221 committed:
fix dataset bug (#5038)

* update lrscheduler

* minor fix

* add pre-commit

* minor fix

* Add __len__ to squad dataset

* minor fix

* Add dureader robust prototype

* dataset implement

* minor fix

* fix var name

* add dureader-yesno train script and dataset

* add readme and fix md5sum

* integrete dureader datasets

* change var names: segment to mode, root to data_file

* minor fix

* update var name

* Fix api bugs

* add dataset readme

* add express ner

* update readme format

* fix format bug

* change readme path

* fix format bug

* fix dataset bug

ZeyuChen committed:
Update README.md

LiuChiachi committed:
Add metric readme doc (#5037)

* add metric readme doc

* add "`" for APIs in doc

* Update README.md

Add "`" for APIs in doc

* Update metrics.md

* Update README.md

* update metric readme

LiuChiachi committed:
Update couplet readme (#5039)

* update couplet readme

* update generation example

ZHUI committed:
Add transformer doc (#5040)

* update transformer doc

* Update transformers.md

* Update transformers.md

* update transformer doc

* Update transformers.md

Steffy-zxf committed:
update paddlenlp text matching docs (#5041)

* add paddle.models api reference docs

* update docs

* update docs

* update docs

* tmp

* update docs

* update docs

* update docs

* update docs

LiuChiachi committed:
Fix TranslationDataset bug (#5043)

* fix translation.py bugs

* delete useless comments

* set couplet download root None

LiuChiachi committed:
fix bugs (#5046)

taixiurong committed:
support xpu in bert (#5044)

Commit:
clean hapi notes, clean eager_run and modify num_labels to num_classes (#5045)

windstamp committed:
bugfix for change arguments use_fp16 to use_amp, use_pure_fp16, multi_precision (#5012)

Steffy-zxf committed:
update docs (#5049)

* add paddle.models api reference docs

* update docs

* update docs

* update docs

* tmp

* update docs

* update docs

* update docs

* update docs

* update docs

* update docs

joey12300 committed:
Add more embedding and sample  for the TokenEmbedding

* add all wiki embedding and part of baidu encyclopedia embedding.

* add embedding example

* add people_daily, weibo, sougou pretrained embedding

* add zhihu, finacial,literature embedding

* Add embedding model readme; add embedding train example and readme

* fix README example

* fix embedding doc

wawltor committed:
update the readme for the word_embedding (#5050)

update the readme for the word_embedding (#5050)

LiuChiachi committed:
Update seq2seq example (#5016)

* update seq2seq, using paddlenlp

* Using new paddlenlp API

* update seq2seqREADME

* wrap dev ds

* delete useless comments

* update predict.py

* using paddlenlp.bleu

* remove shard

* update README, using bleu perl

* delete cand

* Remove tokens that make sentences longer than max_len

* remove pdb

* remove useless code.

* update url and dataset name of vae dataset(ptb and yahoo)

* update seq2seq and vae, data and README

Steffy-zxf committed:
update codes for paddenlp text cls example

update codes for paddenlp text cls example

FrostML committed:
[Transformer] Simplify transformer reader and fix TranslationDataset (#5035)

* fix translation dataset and simplify transformer reader

kinghuin committed:
add ernie_gen for the dygraph mode

add ernie_gen for the dygraph mode

Steffy-zxf committed:
update docs for ernie-tiny

update docs for ernie-tiny

xiemoyuan committed:
Upgrade plato2 using paddle2.0 (#5002)

* The first version of plato2. Not finished the network.

* Update decode stratage.

* Update decode stratage.

* Completed the encoder and decoder. But it will oom.

* Completed the encoder and decoder.

* backend the code.

* Only completed the network of plato2 and nsp.

* Completed the development. But the effect has not be verified.

* Add readme and remove the code about deal PY2 and PY3.

* Modify comment.

* Modify readme and add images.

* Delete the data folder.

Commit:
add glue and electra readme (#5051)

xiemoyuan committed:
Rename 'decoder' to 'decode'. (#5056)

kinghuin committed:
Fix ernie_gen and CRF bug (#5057)

* add ernie_gen

* optimize ernie_gen

* optimize ernie_gen

* optimize ernie_gen code

* fix crf bug

* add ernie_gen __init__.py

* modify nlp version

* fix ernie_gen predict

wawltor committed:
add the support token_embedding of express (#5058)

ZeyuChen committed:
remove examples/bert, unifiy to benchmark/bert

kinghuin committed:
优化ernie gen文档 (#5059)

* add ernie_gen

* optimize ernie_gen

* optimize ernie_gen

* optimize ernie_gen code

* fix crf bug

* add ernie_gen __init__.py

* modify nlp version

* fix ernie_gen predict

* optimize doc

smallv0221 committed:
Add 2 datasets and update squad metric and run_du (#5060)

* update lrscheduler

* minor fix

* add pre-commit

* minor fix

* Add __len__ to squad dataset

* minor fix

* Add dureader robust prototype

* dataset implement

* minor fix

* fix var name

* add dureader-yesno train script and dataset

* add readme and fix md5sum

* integrete dureader datasets

* change var names: segment to mode, root to data_file

* minor fix

* update var name

* Fix api bugs

* add dataset readme

* add express ner

* update readme format

* fix format bug

* change readme path

* fix format bug

* fix dataset bug

* add 2 datasets and update squad metric

wawltor committed:
Add copyright for the word2vec (#5063)

* Add the copyright for the word-embedding

* add the the link for the chinese word embedding

smallv0221 committed:
update dureader-robust readme result (#5064)

* update lrscheduler

* minor fix

* add pre-commit

* minor fix

* Add __len__ to squad dataset

* minor fix

* Add dureader robust prototype

* dataset implement

* minor fix

* fix var name

* add dureader-yesno train script and dataset

* add readme and fix md5sum

* integrete dureader datasets

* change var names: segment to mode, root to data_file

* minor fix

* update var name

* Fix api bugs

* add dataset readme

* add express ner

* update readme format

* fix format bug

* change readme path

* fix format bug

* fix dataset bug

* add 2 datasets and update squad metric

* update readme result

FrostML committed:
alter the default value for vocab (#5062)

smallv0221 committed:
fix bleu bug (#5065)

ceci3 committed:
Add ofa  for bert (#5006)

* fix init

* update

* add ofa for bert

* update

* update

* update

* update readme

* update

* update

* update

FrostML committed:
Update Transformer readme (#5066)

* update readme

* update test=develop

* fix readme

LiuChiachi committed:
Fix seq2seq readme error (#5069)

* update __init__

* update couplet api usage, delete useless import

* update seq2seq readme

* update seq2seq readme path

taixiurong committed:
modify default xpu and gpu to false (#5074)

Steffy-zxf committed:
update codes (#5072)

* update codes

* update codes

* update codes

smallv0221 committed:
fix bleu bug (#5073)

* fix bleu bug

xiemoyuan committed:
Modify readme of Plato2 (#5076)

* delete the images.

* delete the images.

* test

* test

* test

* test

* test

* test

* test

* test

* test

* test

* test

* test

* test

* test

* add imgs.

FrostML committed:
Fix delimiter (#5075)

* fix delimiter

ceci3 committed:
Rename readme (#5080)

* fix init

* update

* update name

smallv0221 committed:
fix dureader-robust readme (#5078)

* fix bleu bug

* fix bleu bug

* fix doc example

* minor fix

* fix squad dataset bug

* fix readme

wawltor committed:
add the reference for the embedding (#5083)

joey12300 committed:
fix bugs;add more baidu embedding name (#5077)

fix bugs;add more baidu embedding name

LiuChiachi committed:
Add vae readme, add infer path (#5088)

* update __init__

* update vae readme

wanghuancoder committed:
mobilenet delete numpy() in print, test=develop (#5061)

taixiurong committed:
modify GPU default config(#5089)

LiuChiachi committed:
Update seq2seq using bleu (#5079)

* update __init__

* update seq2seq example, using bleu

* remove target file arg

* update readme

xiemoyuan committed:
modify readme of plato-2. (#5087)

wawltor committed:
add the benchmark glue for some task (#5086)

* add the benchmark glue for some task

* fix the qnil result for the bert benchmark

* change the from the logger to framework logger

smallv0221 committed:
Add dependency to readme (#5081)

* fix bleu bug

* fix bleu bug

* fix doc example

* minor fix

* fix squad dataset bug

* fix readme

* Add dependency to readme

* minor fix

* change run parameter

kinghuin committed:
modify ernie_gen README (#5090)

modify ernie_gen README (#5090)

FrostML committed:
fix static save (#5091)

ZeyuChen committed:
remove useless readme and example

xiemoyuan committed:
Add data.md (#5071)

* add data.md

* backup code

* Add example for APIs.

* add readme.

* code backup.

* Modify the data.md

* fixed bug.

* Optimize the data.md

xiemoyuan committed:
Del the images of eval. (#5092)

ZeyuChen committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/models into develop

joey12300 committed:
fix url bug for word2vec

fix url bug for word2vec

Commit:
add grad clip to pretrain, modify glue readme and improve glue score … (#5093)

* add grad clip to pretrain, modify glue readme and improve glue score print

* add ernie-2.0 in run_glue.py and readme

Steffy-zxf committed:
update docs (#5095)

* update codes

* update codes

* update codes

* update docs

* update docs

ZeyuChen committed:
Update README.md

ZeyuChen committed:
Update README.md

ZeyuChen committed:
add png photo

ZHUI committed:
add pretrained model reference (#5098)

ZeyuChen committed:
Update README.md

xiemoyuan committed:
modify the readme. (#5099)

wawltor committed:
add the log format for the bert benchmark (#5101)

guoshengCS committed:
Fix link error in PaddleNLP README. (#5100)

* Fix link error in PaddleNLP README.

* Fix link error in README_en.

smallv0221 committed:
fix dureader readme (#5105)

* fix bleu bug

* fix bleu bug

* fix doc example

* minor fix

* fix squad dataset bug

* fix readme

* Add dependency to readme

* minor fix

* change run parameter

* fix readme

Commit:
fix glue readme (#5102)

* fix glue readme

* fix glue readme cased

ZeyuChen committed:
Update README.md

ZeyuChen committed:
Update README_en.md

pangyoki committed:
Add AMP for ResNet200_vd model (#5070)

* add amp for ResNet200_vd

* change use_fp16 to use_amp

* fix script format

* change batch_size

Steffy-zxf committed:
update paddlenlp docs (#5109)

* update codes

* update codes

* update codes

* update docs

* update docs

* update docs

* update docs

wawltor committed:
update the readme for the benchmark bert (#5111)

kinghuin committed:
modify ner readme (#5106)

* modify readme

* modify readme

* modify readme

* modify readme

xiemoyuan committed:
Modify the readme for elmo, dgu and plato2. (#5112)

Commit:
improve example me and delete electra run_glue.py (#5113)

ZeyuChen committed:
re-organize language model and update readme

LiuChiachi committed:
Update readme version info (#5114)

* update __init__

* fix readme version info

ZHUI committed:
fix erniesage for pgl 2.0 alpha 

fix erniesage for pgl 2.0 alpha

smallv0221 committed:
mallv0221 fix readme and change cross-entropy api (#5116)

* fix readme and change cross-entropy api

* change api back

FrostML committed:
work around hung (#5119)

ceci3 committed:
update speed up (#5118)

FrostML committed:
Add transformer-xl for language model (#4987)

* add transformer-xl for language model

wangxicoding committed:
fix bert benchmark fleet compiled (#5117)

LiuChiachi committed:
Update couplet readme (#5120)

* update __init__

* solve conflicts

* update couplet readme

ZHUI committed:
fix for pgl for the erniesage

fix for pgl for the erniesage

joey12300 committed:
Add en vector for the embedding 

* add more English embedding name

* fix doc bug

* delete useless description

* add comments of TokenEmbedding

* add embedding model info

FrostML committed:
Fix hung (#5121)

* fix hung

* add shuffle batch

* update

* reader_seed to shuffle_seed

* seed for shuffle batch

guoshengCS committed:
Add BERT. (#5125)

guoshengCS committed:
Fix single data file in BERT pre-training. (#5127)

FrostML committed:
update reader (#5129)

* shuffle batch

* update reader

* update benchmark

smallv0221 committed:
fix warmup step bug and qa padding (#5128)

guoshengCS committed:
Fix glue set_seed. (#5130)

FrostML committed:
[Perform] Apply pad factor to pad vocab (#5134)

* add pad factor to pad vocab

joey12300 committed:
Fix windows bug (#5140)

* add embedding example in embeddings.md

* add embedding.md link in toke_embedding readme

* fix windows url bug

kinghuin committed:
new chunk_eval (#5138)

* new chunk_eval

* np first

ZeyuChen committed:
Update README.md

smallv0221 committed:
upgrade MapDatasetWrapper (#5132)

* upgrade MapDatasetWrapper

* minor fix

* refine code

FrostML committed:
fix models/dygraph/transformer (#5135)

* fix dygraph

* update decription

wawltor committed:
add the suport the log format for the benchmark (#5153)

add the suport the log format for the benchmark

FrostML committed:
fix reader (#5154)

zhiqiu committed:
fix transformer pe (#5157)

* fix incorrect usage of data loader

* refine code

* remove unused code

* refine code

wawltor committed:
change the format for the bert benchmark (#5160)

change the format for the bert benchmark (#5160)

FrostML committed:
fix single (#5162)

wawltor committed:
Support xpu for ernie and bert in pre-train/glue task (#5173)

* add the xpu support the the paddlenlp

* add the support xpu for the ernie and bert

Co-authored-by: wawltor <fanzeyang0904@hotmail.com>

FrostML committed:
Add static predict (#5147)

* add static predict

* rm position_encoding_init

* fix static predict

wangxicoding committed:
nlp bert benchmark add gradient merge (#5139)

nlp bert benchmark add gradient merge

FrostML committed:
Support pad sequence and batch size can be multiplied by a factor (#5175)

* support pad seq and bsz multi

* alter vocab size

* alter md5sum

* root for reader

FrostML committed:
alter seq pad for default (#5181)

zhangting2020 committed:
set stop_gradient (#5178)

Steffy-zxf committed:
update docs (#5180)

* update codes for senta benchmark

Commit:
Add fleet for transformer benchmark (#5164)

* add fleet, test=develop

windstamp committed:
add dygraph amp support for transformer (#5187)

* add dygraph amp support for transformer

* according params use_amp to decide use dygraph amp

FrostML committed:
Add benchmark readme (#5166)

* add benchmark readme

* add predict for static

* update doc for fleet api

* typo

FrostML committed:
transformer rc0 (#5191)

* add transformer for rc0

* predict no_grad

wzzju committed:
Add pure fp16 training. (#5052)

* add pure fp16 training.

* Add lookup_table in custom_black_list during the pure fp16 training.

kinghuin committed:
fix lac bug (#5190)

fix lac bug (#5190)

kinghuin committed:
add peoplesner, fix msraner (#5183)

add peoplesner, fix msraner (#5183)

kinghuin committed:
fix doc  for the paddlenlp version

* fix doc

* fix version bug

FrostML committed:
Transformer-XL update train (#5200)

* update train

* update eval

FrostML committed:
Change the dir name (#5202)

* change the dir name

* add static step final save

* refine doc

* alter method name

zhangting2020 committed:
support AMP training (#5067)

* support AMP training

FrostML committed:
Refine Transformer-XL readme (#5203)

* fix readme

* update doc

* add / for dir

* refine eval batch size

wzzju committed:
Add new pure fp16 training for ResNet50. (#5047)

FrostML committed:
fix cpu (#5212)

Steffy-zxf committed:
update codes for static and dynamic graph unification (#5215)

* update docs

FrostML committed:
refine cpu (#5214)

* refine transformer cpu

* delete run_pretrain.sh

ZHUI committed:
add gpt2 model for the paddlenlp

add gpt2 model for the paddlenlp

huangjun12 committed:
update doc of video (#5231)

wawltor committed:
fix the pure fp16 of argument (#5236)

fix the pure fp16 of argument (#5236)

FrostML committed:
delete transformer rc0 (#5218)

ZeyuChen committed:
Update README.md

kinghuin committed:
fix crf bug in paddlenlp

fix crf bug in paddlenlp

Steffy-zxf committed:
export text classification model in static graph (#5226)

export text classification model in static graph

Steffy-zxf committed:
Add imdb dataset 

* update docs

* add imdb dataset

* update codes

LiuChiachi committed:
Add CoupletDataset (#5247)

* move coupletdataset to paddlenlp.dataset

* move coupletdataset to paddlenlp.dataset

FrostML committed:
Refine Transformer-xl doc and reader (#5225)

* fix reader

* increase epoch from 100 to 200

ZeyuChen committed:
update README

ZeyuChen committed:
Update exmaple's README

LiuChiachi committed:
Add and fix doc for some paddlenlp datasets (#5249)

* move coupletdataset to paddlenlp.dataset

* move coupletdataset to paddlenlp.dataset

* add doc for coupletdataset copyright, fix other api's sample code

ZeyuChen committed:
update README for 2.0-rc

ZeyuChen committed:
update english readme also

ZeyuChen committed:
update readme, migrad from clear_gradients()->clear_grad()

ZeyuChen committed:
upgrade README

LiuChiachi committed:
delete useless comments (#5258)

ShenYuhan committed:
Update VisualDL doc for embedding (#5259)

fix docs for visualdl

ZeyuChen committed:
remove useless doc

yingyibiao committed:
Support padding direction selection for collate (#5263)

add an optional boolean argument pad_rght to Pad class to support padding direction selection

wangxicoding committed:
Fleet support pure_fp16 in bert benchmark (#5255)

* bert benchmark fleet support pure_fp16

* fix use_pure_fp16

ZHUI committed:
fix package requirement of regex and sentencepiece (#5261)

ZeyuChen committed:
update README

ZHUI committed:
Add checkpoint support for gpt2 model (#5257)

* fix checkpoints problem.

ZeyuChen committed:
fix dead link on README.md

ZeyuChen committed:
fix dead link on README.md

guoshengCS committed:
Add inference support for BERT. (#5199)

* Add inference support for BERT.

* Update inference support for BERT.

Steffy-zxf committed:
Refine text classification and matching examples  (#5251)

* update docs

* update codes

* update docs

* update codes

* update codes

* update codes

* add method tokenizer.get_special_tokens_mask()

ZeyuChen committed:
upgrade model zoo README

ZeyuChen committed:
update model zoo and exmaple readme

guoshengCS committed:
Add amp support for BERT. (#5198)

wuhuachaocoding committed:
add fleet pure_fp16 into train.py (#5268)

FrostML committed:
Transformer dy2sta and inference support (#5209)

* dy2sta and inference support

* delete useless code

* update dir

* sys

* delete useless import

xiemoyuan committed:
Fix DGU bug (#5272)

ZHUI committed:
Quick fix typo (#5273)

LiuChiachi committed:
add seq2seq infer (#5205)

* add seq2seq infer

* update argument description, remove useless import

* add deploy directory

* add deploy directory, add relative import

* update arg usage in README, fix import order.

LiuChiachi committed:
Add some lr schedulers with warmup (#5256)

* add -linear-schedule-with-warmup

* update lr scheduler usage in run_glue

* add import info

* add other 5 scheduler with warmup

* add copyright, update usage in run_glue

* simplify argument, make warmup arg support float and int

* Add some LambdaDecay scheduler with warmup, update usage in run_glue

* update classname and unify two cosine decays into one

* update usage in run_glue

* fix typo

* update WarmUp to Warmup, and update class name about Const, and update doc, and usage

* update usage of decay class

* update usage of decay class

kinghuin committed:
optimize lac and msra_ner example (#5270)

* optimize lac and msra_ner example

* optimize ner and lac

* rename filename to lexical_analysis_dataset_tiny_path0

* modify lac dataset url

LiuChiachi committed:
Add Distill Bi-LSTM (#5158)

* add distill lstm

* update distill and small model

* add data augmentation for distill Bi-LSTM

* update readme

* load embedding from pre-trainined files

* add some dataloader, modify model to support QQP

* update README

* upload senta word dict

* fix qqp distill data loader

* update data augmentation method

* Delete senta_word_dict.txt

* fix chinese data augmentation bug

* update some codes according to review, update readme

* use args, update readme, delete run_bert_finetune.py

* fix utils bugs

* add args

* update distill readme

* update readme

* rename readme's headline

* convert bert to uppercase

wawltor committed:
Add submodule for the PaddleNLP and delete the old directory (#5277)

* remove the old directory for the paddlenlp

* add the submodule for the PaddleNLP

Intsigstephon committed:
Update utility.py (#5279)

huangjun12 committed:
update DataParallel in slowfast (#5298)

* update DP in slowfast and refine nextvlad doc

* refine doc

KPatr1ck committed:
PaddleAudio initial commit (#5299)

KPatr1ck committed:
Refactor code in paddleaudio/models (#5300)

* Refactor code in paddleaudio/models

* Upgrade __getitem__ to support on the fly feature extraction

KPatr1ck committed:
Add sound classification example (#5303)

* Add sound classification example

* Add sound classification example

ZeyuChen committed:
Update PaddleSpeech README

KPatr1ck committed:
Update audio tagging example (#5305)

* Update audio tagging example

* Update audio tagging example

ZeyuChen committed:
update gitmodule

ranchlai committed:
PaddleAudio framework alpha version (#5311)

* added sound classication

* added liscense, clean code, add pre-commit

* update req

* moved to PaddlePaddle-models

* code re-structure

* update README.md

* update README.md

* Update README.md

* add audioset training

* default resample mode to kaiser_fast

* delete some comments

* precommit check

* sha->rev

* add config.ymal

* remove SoundClassification from paddlespeech, since it's in PaddleAudio now

* add labels

* remove old labels

* update code

* empty

* #5300

* add evaluate, etc

* remove trace|

* import evaluate

* path update

* precommit check

* recover slowfast

* restore README.md to paddle:develop

* refactor

* update readme

* update README.md

* refactor

* refactor

* refactor

* refactor

* precommit fixed

* update README.md

* Update README.md

* Update README.md

* Update train.py

changed prefixed, removed some comments

* add wav file for testing

* bug fixed eval,new checkpoint map=0.416

* Update README.md

Co-authored-by: ranchlai <=ranchlai@163.com>

ZeyuChen committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/models into develop

ZeyuChen committed:
Remove submodule of PaddleNLP

ZeyuChen committed:
Add PaddleNLP Research folder for avoid url invalid.

godfanmiao committed:
Update README.md (#5244)

aigcliu committed:
Add SMOKE model (#5308)

* add SMOKE

* add deployment

* add pretrained link.

* Update README.md

* Update README.md

* Update README.md

* update config

* add figure

* fix a typo

* delete unused codes

* add reference links

* resolved problems

* change to 2.1

Co-authored-by: liuyi22 <liuyi22@baidu.com>

KPatr1ck committed:
Add aishell and librispeech dataset (#5312)

* Add aishell and librispeech dataset

* Add aishell and librispeech dataset

* Add aishell and librispeech dataset

* Add UrbanAudioVisualScenes Dataset

* Update features api

ranchlai committed:
Add wav2vec 2.0  (#5313)

* added sound classication

* added liscense, clean code, add pre-commit

* update req

* moved to PaddlePaddle-models

* code re-structure

* update README.md

* update README.md

* Update README.md

* add audioset training

* default resample mode to kaiser_fast

* delete some comments

* precommit check

* sha->rev

* add config.ymal

* remove SoundClassification from paddlespeech, since it's in PaddleAudio now

* add labels

* remove old labels

* update code

* empty

* #5300

* add evaluate, etc

* remove trace|

* import evaluate

* path update

* precommit check

* recover slowfast

* restore README.md to paddle:develop

* refactor

* update readme

* update README.md

* refactor

* refactor

* refactor

* refactor

* precommit fixed

* update README.md

* Update README.md

* Update README.md

* Update train.py

changed prefixed, removed some comments

* add wav file for testing

* bug fixed eval,new checkpoint map=0.416

* Update README.md

* added dcase task1b example

* update README.md

* code fixed for last review

* fixed level string formating

* fixed according to PR reviews

* added wav2vec2.0

* restore datatsets

* add liscense, remove scipy, move test_audio to cloud

* remove 3rd-party dependency:pathos

* add testing for wav2vec2

* update README.md

* updated README.md, added librispeech results

* Revert "updated README.md, added librispeech results"

This reverts commit da4012958e8e0bf2d7f4b608f74518583dd7d73b.

* code fixed from reviews

* add librispeech test

* remove pathos imports

* updated README.md

* update README.md

* minor-fix according to code reviews

* updated README_LP.MD

* fixed according to code review

* fixed according to code review

* added preprocessing example

* removed dcase2021_task1b from examples

* remove preprocessing from examples

* added amsoftmax to losses

* added eer/min_dcf to metrics

* updated __init__.py

Co-authored-by: ranchlai <=ranchlai@163.com>

ranchlai committed:
Add transform/functional to paddleaudio.  (#5319)

* added sound classication

* added liscense, clean code, add pre-commit

* update req

* moved to PaddlePaddle-models

* code re-structure

* update README.md

* update README.md

* Update README.md

* add audioset training

* default resample mode to kaiser_fast

* delete some comments

* precommit check

* sha->rev

* add config.ymal

* remove SoundClassification from paddlespeech, since it's in PaddleAudio now

* add labels

* remove old labels

* update code

* empty

* #5300

* add evaluate, etc

* remove trace|

* import evaluate

* path update

* precommit check

* recover slowfast

* restore README.md to paddle:develop

* refactor

* update readme

* update README.md

* refactor

* refactor

* refactor

* refactor

* precommit fixed

* update README.md

* Update README.md

* Update README.md

* Update train.py

changed prefixed, removed some comments

* add wav file for testing

* bug fixed eval,new checkpoint map=0.416

* Update README.md

* added dcase task1b example

* update README.md

* code fixed for last review

* fixed level string formating

* fixed according to PR reviews

* added wav2vec2.0

* restore datatsets

* add liscense, remove scipy, move test_audio to cloud

* remove 3rd-party dependency:pathos

* add testing for wav2vec2

* update README.md

* updated README.md, added librispeech results

* Revert "updated README.md, added librispeech results"

This reverts commit da4012958e8e0bf2d7f4b608f74518583dd7d73b.

* code fixed from reviews

* add librispeech test

* remove pathos imports

* updated README.md

* update README.md

* minor-fix according to code reviews

* updated README_LP.MD

* fixed according to code review

* fixed according to code review

* added preprocessing example

* removed dcase2021_task1b from examples

* remove preprocessing from examples

* added amsoftmax to losses

* added eer/min_dcf to metrics

* updated __init__.py

* add stft,spectrogram, melspectrogram, log-melspectrogram

* add _internal, transoform, functional to imports

* add new module: functional

* add new module: window.py to _internel/

* add correspoding new unit-test for the new modules

* added ISTFT

* clean code and docstring, clean unit test

* clean code and docstring

* functional

* added back preprocessing

* add README.md

* remove preprocessing for now

* clean code, add doc

* change _internal to signal

* add new transoforms

* add new functionals

* add eps to amsoftmax, return the prediction

* add ffmpeg backend

* remove dithering in depth-convert, add ffmpeg to backend

* add Mudecode/enccode/RandomCodec

* changed variable name, fixed bug

* use namedtuple for returning

* refactor utils

* refactor

* add melspectrogram/spectrogram, add doc string

* add doc string, clean code

* rename window to windowing

* updated docstring, minor bug fixed

* move losses.py to future examples

* remove mu_encode/decode

* refactor

* move metrics to future examples

* remove features/

* naming changes for mu law algorithms

* update test, add testing utils

* fixed import

ranchlai committed:
Update doc string with examples/shapes, controllable backends, and some bug fixed (#5324)

* added sound classication

* added liscense, clean code, add pre-commit

* update req

* moved to PaddlePaddle-models

* code re-structure

* update README.md

* update README.md

* Update README.md

* add audioset training

* default resample mode to kaiser_fast

* delete some comments

* precommit check

* sha->rev

* add config.ymal

* remove SoundClassification from paddlespeech, since it's in PaddleAudio now

* add labels

* remove old labels

* update code

* empty

* #5300

* add evaluate, etc

* remove trace|

* import evaluate

* path update

* precommit check

* recover slowfast

* restore README.md to paddle:develop

* refactor

* update readme

* update README.md

* refactor

* refactor

* refactor

* refactor

* precommit fixed

* update README.md

* Update README.md

* Update README.md

* Update train.py

changed prefixed, removed some comments

* add wav file for testing

* bug fixed eval,new checkpoint map=0.416

* Update README.md

* added dcase task1b example

* update README.md

* code fixed for last review

* fixed level string formating

* fixed according to PR reviews

* added wav2vec2.0

* restore datatsets

* add liscense, remove scipy, move test_audio to cloud

* remove 3rd-party dependency:pathos

* add testing for wav2vec2

* update README.md

* updated README.md, added librispeech results

* Revert "updated README.md, added librispeech results"

This reverts commit da4012958e8e0bf2d7f4b608f74518583dd7d73b.

* code fixed from reviews

* add librispeech test

* remove pathos imports

* updated README.md

* update README.md

* minor-fix according to code reviews

* updated README_LP.MD

* fixed according to code review

* fixed according to code review

* added preprocessing example

* removed dcase2021_task1b from examples

* remove preprocessing from examples

* added amsoftmax to losses

* added eer/min_dcf to metrics

* updated __init__.py

* add stft,spectrogram, melspectrogram, log-melspectrogram

* add _internal, transoform, functional to imports

* add new module: functional

* add new module: window.py to _internel/

* add correspoding new unit-test for the new modules

* added ISTFT

* clean code and docstring, clean unit test

* clean code and docstring

* functional

* added back preprocessing

* add README.md

* remove preprocessing for now

* clean code, add doc

* change _internal to signal

* add new transoforms

* add new functionals

* add eps to amsoftmax, return the prediction

* add ffmpeg backend

* remove dithering in depth-convert, add ffmpeg to backend

* add Mudecode/enccode/RandomCodec

* changed variable name, fixed bug

* use namedtuple for returning

* refactor utils

* refactor

* add melspectrogram/spectrogram, add doc string

* add doc string, clean code

* rename window to windowing

* updated docstring, minor bug fixed

* move losses.py to future examples

* remove mu_encode/decode

* refactor

* move metrics to future examples

* remove features/

* naming changes for mu law algorithms

* update test, add testing utils

* fixed import

* fixed import

* fixed duplicate output in logging

* add code examples, shape info, etc

* add doc for public functions

* make backend controllable

* fixed coding stype in docstring

ranchlai committed:
Export parameters for setting power_to_db in LogMel transform (#5326)

* added sound classication

* added liscense, clean code, add pre-commit

* update req

* moved to PaddlePaddle-models

* code re-structure

* update README.md

* update README.md

* Update README.md

* add audioset training

* default resample mode to kaiser_fast

* delete some comments

* precommit check

* sha->rev

* add config.ymal

* remove SoundClassification from paddlespeech, since it's in PaddleAudio now

* add labels

* remove old labels

* update code

* empty

* #5300

* add evaluate, etc

* remove trace|

* import evaluate

* path update

* precommit check

* recover slowfast

* restore README.md to paddle:develop

* refactor

* update readme

* update README.md

* refactor

* refactor

* refactor

* refactor

* precommit fixed

* update README.md

* Update README.md

* Update README.md

* Update train.py

changed prefixed, removed some comments

* add wav file for testing

* bug fixed eval,new checkpoint map=0.416

* Update README.md

* added dcase task1b example

* update README.md

* code fixed for last review

* fixed level string formating

* fixed according to PR reviews

* added wav2vec2.0

* restore datatsets

* add liscense, remove scipy, move test_audio to cloud

* remove 3rd-party dependency:pathos

* add testing for wav2vec2

* update README.md

* updated README.md, added librispeech results

* Revert "updated README.md, added librispeech results"

This reverts commit da4012958e8e0bf2d7f4b608f74518583dd7d73b.

* code fixed from reviews

* add librispeech test

* remove pathos imports

* updated README.md

* update README.md

* minor-fix according to code reviews

* updated README_LP.MD

* fixed according to code review

* fixed according to code review

* added preprocessing example

* removed dcase2021_task1b from examples

* remove preprocessing from examples

* added amsoftmax to losses

* added eer/min_dcf to metrics

* updated __init__.py

* add stft,spectrogram, melspectrogram, log-melspectrogram

* add _internal, transoform, functional to imports

* add new module: functional

* add new module: window.py to _internel/

* add correspoding new unit-test for the new modules

* added ISTFT

* clean code and docstring, clean unit test

* clean code and docstring

* functional

* added back preprocessing

* add README.md

* remove preprocessing for now

* clean code, add doc

* change _internal to signal

* add new transoforms

* add new functionals

* add eps to amsoftmax, return the prediction

* add ffmpeg backend

* remove dithering in depth-convert, add ffmpeg to backend

* add Mudecode/enccode/RandomCodec

* changed variable name, fixed bug

* use namedtuple for returning

* refactor utils

* refactor

* add melspectrogram/spectrogram, add doc string

* add doc string, clean code

* rename window to windowing

* updated docstring, minor bug fixed

* move losses.py to future examples

* remove mu_encode/decode

* refactor

* move metrics to future examples

* remove features/

* naming changes for mu law algorithms

* update test, add testing utils

* fixed import

* fixed import

* fixed duplicate output in logging

* add code examples, shape info, etc

* add doc for public functions

* make backend controllable

* fixed coding stype in docstring

* export parameters for power_to_db LogMel transform

* default to_db to False to be consistent with functional

* fixed typo in docstring

KPatr1ck committed:
Update usage of new APIs (#5331)

ranchlai committed:
update example to use new functionals (#5330)

ranchlai committed:
Update PaddleAudio transforms and functionals (#5334)

* added reverb/noisify/AudioReader/RandomChoice/RandomApply

* bug fixed

* transform name changes

* work around for bug in paddle's groupnorm

* upgraded to use float64 inside for high numerical acc

* fixed docstring, add nn.Layer as super for Noisify

* fixed docstring

* added mfcc func/trans and dct function

* updated unit test

* add dtype to control datatype in win function

* add dtype control in transforms

* add dtype control in functionals

* updated test

* added dtype control, updated test

ranchlai committed:
Add Speaker model examples (#5337)

* added speaker

* update readme

TingquanGao committed:
fix: fix the invalid link (#5347)

littletomatodonkey committed:
add 1st reprod doc (#5352)

littletomatodonkey committed:
fix doc (#5354)

dyning committed:
Update ThesisReproduction_CV.md

dyning committed:
updata ThesisReproduction_CV (#5355)

littletomatodonkey committed:
fix readme (#5356)

* fix readme

* fix reprod

* Update ThesisReproduction_CV.md

* fix number

* add grad print example

* add content

* add content

* fix

dyning committed:
Update ThesisReproduction_CV.md

littletomatodonkey committed:
add FAQ in chapter 4. (#5358)

* add ch4

* Update ThesisReproduction_CV.md

* Update ThesisReproduction_CV.md

* Update ThesisReproduction_CV.md

littletomatodonkey committed:
fix doc (#5359)

littletomatodonkey committed:
add new faq for jingying-plan (#5367)

* add new faq for jingying-plan

* add set device

* fix faq
